### PR TITLE
Add update php repo github action

### PR DESCRIPTION
**Pull request by deannagarcia**
This Github action will automatically sync to the php repository whenever we push a new release. It will copy over the new files and add proper tags.

### CODE REVIEW

File: .github/workflows/update_php_repo.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: Update protobuf-php Repo
+
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+
```
Comment (zhangskz) [inline .github/workflows/update_php_repo.yml:6]: I think this needs to be expanded to include RCs. I also suspect the "."  should also be escaped.

File: .github/workflows/update_php_repo.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      - name: Copy files
+        run: |
+          rm -rf src
+          cp -r protobuf/php/src .
+          cp protobuf/php/composer.json.dist composer.json
+          rm -rf protobuf
+      - name: Push Changes
+        run: |
+          git add .
+          git commit -m "$VERSION sync"
```
Comment (zhangskz) [inline .github/workflows/update_php_repo.yml:42]: Nit: You can do `git commit -a -m "$VERSION sync" instead.

**Review by zhangskz — APPROVED** [review]

File: .github/workflows/update_php_repo.yml (hunk 1 above)
Comment (deannagarcia) [reply .github/workflows/update_php_repo.yml:6]: Added a new tag to match on RCs.

File: .github/workflows/update_php_repo.yml (hunk 2 above)
Comment (deannagarcia) [reply .github/workflows/update_php_repo.yml]: Fixed.

**Review by zhangskz — APPROVED** [review]

**Review by zhangskz — APPROVED** [review]

**Comment by googleberg** [comment]
These changes have landed through copybara. Copybara should have autoclosed or automerged this. Closing manually.

**Pull request closed without merging**